src/common/options.cc

```diff
@@ -7683,6 +7683,10 @@ static std::vector<Option> get_rbd_options() {
     .set_min(1)
     .set_description("minimum schedule tick (in milliseconds) for QoS"),
 
+    Option("rbd_qos_skip_discard", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
```

Comment:
This potentially seems a little too unique to your specific use-case / hackish. If the goal is to still issue massive discards w/ `mkfs` (instead of disabling it) in combination with QoS BPS limits, perhaps it should be a config item with a list of OPs to exclude instead of being explicit for just discard for your one use-case.

Reply:
Updated with common solution, a config item with a list of OPs to exclude QoS.
In the situation several containers run in one node for different services with intensive data write and rm, discard is necessary to get backend storage space oversold, and BPS QoS is also necessary to avoid one service using up the whole network bandwidth.